src/main/interface/settings.c

```diff
@@ -386,6 +386,9 @@ const clivalue_t valueTable[] = {
     { "acc_lpf_hz",                 VAR_UINT16 | MASTER_VALUE, .config.minmax = { 0, 400 }, PG_ACCELEROMETER_CONFIG, offsetof(accelerometerConfig_t, acc_lpf_hz) },
     { "acc_trim_pitch",             VAR_INT16  | MASTER_VALUE, .config.minmax = { -300, 300 }, PG_ACCELEROMETER_CONFIG, offsetof(accelerometerConfig_t, accelerometerTrims.values.pitch) },
     { "acc_trim_roll",              VAR_INT16  | MASTER_VALUE, .config.minmax = { -300, 300 }, PG_ACCELEROMETER_CONFIG, offsetof(accelerometerConfig_t, accelerometerTrims.values.roll) },
+    { "acczero_x",                  VAR_INT16  | MASTER_VALUE, .config.minmax = { INT16_MIN, INT16_MAX }, PG_ACCELEROMETER_CONFIG, offsetof(accelerometerConfig_t, accZero.raw[X]) },
```

Comment:
I am wondering if we should make these to reflect the fact that they are calibration data?

Reply:
I'm open to suggestions, but I was just replicating the way the magnetometer calibration is represented.  `magzero_x`, etc.

Reply:
Alternately we could use the `MODE_ARRAY` type like:

`    { "acc_calibration",            VAR_INT16  | MASTER_VALUE | MODE_ARRAY, .config.array.length = 3, PG_ACCELEROMETER_CONFIG, offsetof(accelerometerConfig_t, accZero.raw) },
`

```
# get acc_calibration
acc_calibration = 789,-46,-445
Array length: 3
```

Simplifies it down to a single entry in the diff and this is not a value we expect people to set manually.  If we go this route should probably change mag calibration to be consistent.

Reply:
I like the array representation. My thinking is more that if users store a diff, and these values show up, they might think they are there in error since they didn't set them, and delete / not restore them. If they are called `acc_calibration`, `mag_calibration` that at least gives them the idea that they were generated by the calibration.

Reply:
Well it looks like there's a secondary bug that will block the array approach - or at least defeat the goal of the change.

Just tested and saw strange results in the `diff` output.  Turns out that the `diff` appears to only be checking the first element of the array against defaults to determine if the values should be output.

So `acc_calibration = 0,-140,352` won't show up in a `diff`.

So I guess I'll look into that bug first...